client/app/web/modules/series/root/detailSerie/Info.js

```diff
@@ -1,10 +1,15 @@
-import React from 'react' 
+import React from 'react';
+import infoStyles from './info.scss';
 
 const Info = ({ serie }) =>
-    <div>
-        <img src={ serie.image.original } alt={ serie.title }/>
-        <h3>{ serie.title }</h3>
-        <span>{ serie.languaje }</span>
+    <div className={ infoStyles.info }>
+        <div>
+            <img src={ serie.image.medium } alt={ serie.title }/>
+        </div>
+        <div className={ infoStyles.detail }>
+            <h3>{ serie.title }</h3>
+            <span>{ serie.languaje }</span>
```

Comment:
languaGe

Reply:
This was a problem in the generation of data, we have to define the definitive model with al data that we want to show.